pkg/action/install.go

```diff
@@ -291,7 +300,7 @@ func (i *Install) Run(chrt *chart.Chart, vals map[string]interface{}) (*release.
 	// At this point, we can do the install. Note that before we were detecting whether to
 	// do an update, but it's not clear whether we WANT to do an update if the re-use is set
 	// to true, since that is basically an upgrade operation.
-	if _, err := i.cfg.KubeClient.Create(resources); err != nil {
+	if _, err := i.cfg.KubeClient.Update(adoptedResources, resources, false); err != nil {
```

Comment:
if `adoptedResources` is empty, should we revert back to a `Create` operation here?